CFC_DataCollector/recommender/alternative_trips_module.py

```diff
@@ -0,0 +1,105 @@
+import get_trips
+from common import store_trip_in_db, find_perturbed_trips
+from trip import E_Mission_Trip
+from get_database import get_alternative_trips_db, get_perturbed_trips_db
+
+#import Profiles
+
+"""
+
+High level overivew of alternative_trips_pipeline
+
+The main method for this pipeline is calc_perturbed_trips
+We first construct a trip iterator, and look at each trip where the pipelineFlags attribte "alternativesStarted" is False
+For each of these trips, we create an array of trip objects, which are modifications to the original trip object with perturbed times
+We also have a database collection that associates each original trip, with all of its perturbed trips
+
+For each array of perturbed trip objects, we schedule those queries via CRON jobs
+We initialize each perturbed trip as None, and we update the collection as the queries are being made
+
+
+Overview of helper files relevant to this pipeline:
+    -query.py -> makes a google maps query immediately to get the directions
+              -> will also store the query into the database collection
+    -perturb.py -> schedules calls to query.py on cron jobs
+
+    @TODO: this doesn't exist yet, but it would make more sense than to put it in here in order to keep the code clean
+    -database_util.py -> contains all the helper methods to abstract out interaction with the database
+
+
+"""
+
+
+# Invoked in recommendation pipeline to get perturbed trips user should consider
+def calc_alternative_trips(trip_iterator):
+    while (trip_iterator.next() != None):
+        curr_trip = trip_iterator.next()
+        if not curr_trip.getPipelineFlags().alternativesStarted:
+            curr_trip.getPipelineFlags().startAlternatives()
+            curr_unique_id = curr_trip._id
+            list_of_perturbed_trips = find_perturbed_trips(curr_unique_id)
+            
+            #@TODO: need a method that initializes the relationship bewteen a trip, and its perturbations
```

Comment:
@joshzarrabi, @jeffdh5, who is going to address this TODO? Is it really required?